src/main/kotlin/me/parker/nextstep/kotlinlotto/controller/LottoController.kt

```diff
@@ -0,0 +1,22 @@
+package me.parker.nextstep.kotlinlotto.controller
+
+import me.parker.nextstep.kotlinlotto.domain.LottoTicket
+import me.parker.nextstep.kotlinlotto.domain.LottoWinningMachine
+import me.parker.nextstep.kotlinlotto.view.ConsoleInput
+import me.parker.nextstep.kotlinlotto.view.ConsoleResult
+
+fun main(args: Array<String>) {
```

Comment:
main 함수의 명령 파라미터는 생략도 가능하겠네요! 👍 
```suggestion
fun main() {
```